### PR TITLE
GUI fixes and solution release/debug fix

### PR DESCRIPTION
**Pull request by Telanor**
Just a small change to make the UI look better for windows 7 users.  Also some of the projects in the solution only had debug builds, so I added release configurations for them

### CODE REVIEW

**Comment by politas** [comment]
I'm guessing the csproj files are used by Visual Studio for building? Our primary build method is make on linux, which would explain the lack, I suppose.

File: GUI/Main.Designer.cs [hunk 1]
```diff
@@ -981,12 +980,12 @@ private void InitializeComponent()
             // KSPVersionLabel
             // 
             this.KSPVersionLabel.Anchor = ((System.Windows.Forms.AnchorStyles)((System.Windows.Forms.AnchorStyles.Top | System.Windows.Forms.AnchorStyles.Right)));
-            this.KSPVersionLabel.AutoSize = true;
-            this.KSPVersionLabel.Location = new System.Drawing.Point(862, 10);
+            this.KSPVersionLabel.Location = new System.Drawing.Point(788, 10);
             this.KSPVersionLabel.Name = "KSPVersionLabel";
-            this.KSPVersionLabel.Size = new System.Drawing.Size(146, 13);
+            this.KSPVersionLabel.Size = new System.Drawing.Size(227, 13);
```
Comment (politas) [inline GUI/Main.Designer.cs]: This has been bugging me, too! Thanks for tidying that up!

**Comment by politas** [comment]
I can see no visual difference on my Linux system, so this doesn't appear to be harmful to non-Win10 users. Can you post an image showing the difference it makes for you?

**Comment by Telanor** [comment]
Before: 
before

After:
after

Mostly just affected the buttons on various forms

**Comment by politas** [comment]
Ah, I see. I wonder if there's a way to get Mono to use a different theme on Linux? The flat theme looks much better there, actually.

**Comment by Telanor** [comment]
Well the easy way would be to replace all of the FlatStyle.Flat constants with a function that returns a value based on OS, but the VS designer will just replace them all the second you open up a form.

What does it look like on linux if you go into the Program.cs and disable `Application.EnableVisualStyles();`?

**Comment by politas** [comment]
That appears to again use the system default style.

**Comment by politas** [comment]
Now that I notice the differences, I am left wondering why the flat design was chosen rather than the system design. It's clearly a decision that was made, or all the fields would not be labelled as `*.FlatStyle = System.Windows.Forms.FlatStyle.Flat` - it gives us a consistent UI across platforms, and a modern one. Given the cross-platform nature of CKAN, I'm very hesitant to make sweeping UI changes that may significantly affect users on minority platforms, especially given the regular reports of GUI problems on Macs we're already getting. Could I suggest you consider helping out with #1840, instead?

**Comment by politas** [comment]
Have decided to stick with the flat design and not make things confusing. We will possibly be completely overhauling the GUI in the near future following #1840 Cherry-picked the KSPVersionLabel fix with thanks.

**Comment by Telanor** [comment]
That's fine, I just figured I'd do a pull request since I'd made the
changes for myself anyway

**Comment by politas** [comment]
And  we really do appreciate it, even when we don't end up using it!

**Pull request closed without merging**